github/orgs_projects.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // ListProjectsOptions specifies the optional parameters to the
 // OrganizationsService.ListProjects method.
-type ListProjectsOptions struct {
+type ProjectsListOptions struct {
```

Comment:
`ProjectListOptions` sounds better than `ProjectsListOptions`.

Also, you'll need to update the comment to use the new name.